packages/common/src/index.ts

```diff
@@ -127,7 +127,7 @@ export interface CommonOpts extends BaseOpts {
    * either from a chain directly supported or a custom chain
    * passed in via {@link CommonOpts.customChains}.
    */
-  chain: string | number | Chain | BN | object
+  chain: string | number | Chain | bigint | object
```

Comment:
Just for the record (nothing to be changed here imminently): these kind of API parts are very much like the `BNLike` ones and at the end we want to remain consistent here. So we either remove BN - as it is done here - or also keep like in `BigIntLike` right now.

But I guess the overall API needs some final consistency looks anyhow once the major work is done.

(side note: my own tendency towards removing `BN.js` completely is actually growing now seeing all this fresh air coming in. 😋 Why wait another year and *then* put the burden on the users to make this transition when we can very well do it now. And since many return values are now getting switched to `BigInt` anyhow, the argument "maybe someone wants to stay on BN.js" is also very much de-valued. Just not really possible anyhow."